build.assets/Dockerfile-centos6

```diff
@@ -7,7 +7,9 @@ ENV LANGUAGE=en_US.UTF-8 \
     LC_ALL=en_US.UTF-8 \
     LC_CTYPE=en_US.UTF-8
 
-RUN yum makecache fast && \
+RUN sed -i 's%^mirrorlist%#mirrorlist%g' /etc/yum.repos.d/* && \
+    sed -i 's%#baseurl=http://mirror.centos.org%baseurl=http://vault.centos.org%g' /etc/yum.repos.d/* && \
```

Comment:
Nit: I would add a comment explaining why we're replacing the repo here and in Dockerfile below as well.